tests/test_actual_langservers.py

```diff
@@ -322,8 +379,7 @@ def do_bar():
             assert diag_msgs == [
                 "Non-void function does not return a value",
                 "Use of undeclared identifier 'do_'",
-                "Expected '}'\n\nfoo.c:9:16: note: to match this '{'",
-                "To match this '{'\n\nfoo.c:9:37: error: expected '}'",
+                "Expected '}'",
```

Comment:
No idea why this changes

Reply:
If I recall correctly - that's a result of sending client's capabilities during the server initialization